Bio/SeqIO/FastaIO.py

```diff
@@ -137,7 +139,12 @@ def FastaTwoLineParser(handle):
 class FastaIterator(SequenceIterator):
     """Parser for Fasta files."""
 
-    def __init__(self, source, alphabet=None, title2ids=None):
+    def __init__(
+        self,
+        source: Union[str, TextIO],
+        alphabet=None,
+        title2ids: Optional[Callable[[str], str]] = None,
```

Comment:
It seems that the type hint here is incorrect (based on the explanation in the docstrings).

I also see that the signature of `title2ids` is repeated in multiple places. Would it be an idea to create an alias for this instead? Something like:

```
Title2IDsFunc = Callable[[str], Tuple[str, str, str]]
```

Reply:
Fixed, thanks!